### PR TITLE
Hotjar Documentation Update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added `track` call documentation and updated the connection settings.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/analytics/hotjar.mdx [hunk 1]
```diff
@@ -35,29 +35,46 @@ Follow our guide on <a href="https://rudderstack.com/docs/connections/adding-sou
...
+- **OneTrust Cookie Category**: Specify the OneTrust category name for mapping the OneTrust consent settings to RudderStack's consent purposes.
+
+<div class="infoBlock">
+
+For more information on this setting, refer to the <a href="https://rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/consent-managers/onetrust/">OneTrust Consent Manager</a> guide.
+</div>
+
+Finally, click on **Next** to complete the configuration. Hotjar will now be added and enabled as a destination in RudderStack.
 
 ## Sending events to Hotjar
```
Comment (ameypv-rudder) [inline destinations/analytics/hotjar.mdx:55]: Waiting for confirmation on this before updating the section. @akashrpo

File: destinations/analytics/hotjar.mdx [hunk 2]
```diff
@@ -35,29 +35,47 @@ Follow our guide on <a href="https://rudderstack.com/docs/connections/adding-sou
...
+
+## Track
 
-There is **no need** to explicitly call any of the `page` or `track` methods as Hotjar auto-tracks your user data.
+The `track` call leverages Hotjar's [**Events API**](https://help.hotjar.com/hc/en-us/articles/4405109971095) to record the user events along with any associated properties.
+
+A sample `track` call is as shown below:
+
+```javascript
+rudderanalytics.track("sample_event");
```
Comment (akashrpo) [inline destinations/analytics/hotjar.mdx:62]:
```suggestion
rudderanalytics.track("custom_event");
```
Let's update it to custom_event as the second parameter in Hotjar is the custom name

File: destinations/analytics/hotjar.mdx [hunk 3]
```diff
@@ -35,29 +35,47 @@ Follow our guide on <a href="https://rudderstack.com/docs/connections/adding-sou
...
+
+For more information on this setting, refer to the <a href="https://rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/consent-managers/onetrust/">OneTrust Consent Manager</a> guide.
+</div>
+
+Finally, click on **Next** to complete the configuration. Hotjar will now be added and enabled as a destination in RudderStack.
+
+## Track
 
-There is **no need** to explicitly call any of the `page` or `track` methods as Hotjar auto-tracks your user data.
+The `track` call leverages Hotjar's [**Events API**](https://help.hotjar.com/hc/en-us/articles/4405109971095) to record the user events along with any associated properties.
```
Comment (akashrpo) [inline destinations/analytics/hotjar.mdx:57]: Since there are no properties being passed in track. Let's modify this.

Comment (ameypv-rudder) [reply destinations/analytics/hotjar.mdx]: this is done.

**Review by akashrpo — APPROVED** [review]
LGTM

File: destinations/analytics/hotjar.mdx [hunk 4]
```diff
@@ -35,29 +35,47 @@ Follow our guide on <a href="https://rudderstack.com/docs/connections/adding-sou
...
 
 ### Connection settings
 
 To successfully configure Hotjar as a destination in RudderStack, enter the following connection settings:
 
-- Enter your Hotjar **Site ID**. You can get your site ID by logging into your Hotjar account and navigating to **Settings** - **Sites & Organizations**. 
+- **Site ID**: Enter your Hotjar site ID. You can get your site ID by logging into your Hotjar account and navigating to **Settings** - **Sites & Organizations**.
 
-Click on **Next** to complete the configuration. Hotjar will now be added and enabled as a destination in RudderStack.
+- **OneTrust Cookie Category**: Specify the OneTrust category name for mapping the OneTrust consent settings to RudderStack's consent purposes.
```
Comment (saikumarrs) [inline destinations/analytics/hotjar.mdx:46]: @ameypv-rudder if we start adding this info, we should do for all the destinations. However, as it is the same for all of them, I would not do that. This should be driven more from the JavaScript source and we've already added that.

Comment (ameypv-rudder) [reply destinations/analytics/hotjar.mdx]: Makes sense. Removed this section.

**Pull request merged**